spidermon/contrib/scrapy/monitors/monitors.py

```diff
@@ -417,9 +420,17 @@ def run(self, result):
             raise NotConfigured(
                 "To enable field coverage monitor, set SPIDERMON_ADD_FIELD_COVERAGE=True in your project settings"
             )
+
         return super().run(result)
 
     def test_check_if_field_coverage_rules_are_met(self):
+        skip_no_items = self.crawler.settings.get(
```

Comment:
**suggestion:** We should use `.getbool()` here instead, so we can parse boolean-like strings as valid values too. That way, we'll support setting this value in ScrapyCloud or the command line.